omod/src/main/webapp/pages/login.gsp

```diff
@@ -41,6 +37,15 @@ ${ ui.includeFragment("referenceapplication", "infoAndErrorMessages") }
             updateSelectedOption();
         });
         
+        jQuery('#login-form').submit(function(e) {
+        	var sessionLocationVal = jQuery('#sessionLocationInput').val();
+        	
+        	if (!sessionLocationVal) {
```

Comment:
@dkayiwa i don't understand the question. Line 29 should always be executed when updateSelectedOption() is called, the condition makes sure that a location was actually chosen. Is that what you were asking?